src/.nuspec/Uno.Resizetizer.windows.skia.targets

```
@@ -21,10 +21,12 @@
 	</Target>
 
 	<Target
-			Name="GenerateUnoSplashWindowsSkia"
-			AfterTargets="GenerateUnoSplashScreens"
-			DependsOnTargets="GenerateUnoSplashScreens"
+			Name="GenerateUnoSplashWindowsSkia"			
+			BeforeTargets="ProcessUnoSplashScreens"
 			Condition="'$(DesignTimeBuild)' != 'True' And '@(UnoSplashScreen)' != ''">
+			Inputs="$(_UnoSplashInputsFile);@(UnoSplashScreen)"
+			Outputs="$(_UnoSplashStampFile)">
```

This will failt to compile

```suggestion
			Name="GenerateUnoSplashWindowsSkia"			
			BeforeTargets="ProcessUnoSplashScreens"
			Condition="'$(DesignTimeBuild)' != 'True' And '@(UnoSplashScreen)' != ''"
			Inputs="$(_UnoSplashInputsFile);@(UnoSplashScreen)"
			Outputs="$(_UnoSplashStampFile)">
```